docs/src/index.md

```diff
@@ -1,8 +1,61 @@
 # TerminalLoggers.jl
 
-```@index
+TerminalLoggers provides a logger type [`TerminalLogger`](@ref) which can
+format your log messages in a richer way than the default `ConsoleLogger` which
+comes with the julia standard `Logging` library.
+
+
+## Installation and setup
+
+```julia-repl
+pkg> add https://github.com/c42f/TerminalLoggers.jl
 ```
 
-```@autodocs
-Modules = [TerminalLoggers]
+To use `TerminalLogger` in all your REPL sessions by default, you may add a
+snippet such as the following to your startup file (in `.julia/config/startup.jl`)
+
+```julia
+atreplinit() do repl
+    try
+        @eval begin
+            import Logging: global_logger
+            import TerminalLoggers: TerminalLogger
+            global_logger(TerminalLogger(stderr))
+        end
+    catch
+    end
+end
 ```
+
+
+## Progress bars
+
+`TerminalLogger` displays progress logging as a set of progress bars which are
+cleanly separated from the rest of the program output at the bottom of the
+terminal. Try an example like the following
+
+```julia
+global_logger(TerminalLogger(stderr, right_justify=120))
+for i=1:100
+    @info "Some progress" progress=i/100
```

Comment:
Strictly speaking, it's more "correct" to use `progress=(i-1)/100`, isn't it? Or maybe move this after `sleep(0.01)`?

This sometimes leads to a bit annoying situation where the progress bar says 100% but the last job is not quite done yet.

Reply:
Definitely. Will fix.

I feel like we should modify this shortly to use your new progress type anyway :-)